File: java/test/PaintCalculatorTest.java
import static org.junit.Assert.*;
import static org.hamcrest.CoreMatchers.is;

import org.junit.Test;


public class PaintCalculatorTest {

	@Test
	public void testCalculateCeilingOfSquareRoom(){
		PaintCalculator pc = new PaintCalculator(new SquareRoom(10,10));
		assertThat(pc.getGallonsOfPaintForCeiling(), is(1));
	}

	@Test
	public void testCalculateCeilingOfLargeSquareRoom(){
		PaintCalculator pc = new PaintCalculator(new SquareRoom(50,50));
		assertThat(pc.getGallonsOfPaintForCeiling(), is(8));
	}
	@Test
	public void testCalculateCeilingOfCircularRoom(){
		PaintCalculator pc = new PaintCalculator(new CircularRoom(10.0));
		assertThat(pc.getGallonsOfPaintForCeiling(), is(1));

		
	}
}

File: ruby/test/paint_calculator_test.rb
require 'test_helper'
class PaintCalculatorTest < Test::Unit::TestCase

  def test_calculate_square_room
    paint_calculator = PaintCalculator.new(SquareRoom.new(10.0,10.0))
    assert_equal(1, paint_calculator.gallons_of_paint_for_ceiling)
  end

  def test_calculate_large_square_room
    paint_calculator = PaintCalculator.new(SquareRoom.new(50.0,50.0))
    assert_equal(8, paint_calculator.gallons_of_paint_for_ceiling)
  end

  def test_calculate_circular_room
    paint_calculator = PaintCalculator.new(CircularRoom.new(10.0))
    assert_equal(1, paint_calculator.gallons_of_paint_for_ceiling)
  end
end
